docker-compose.yaml

```diff
@@ -1,56 +1,54 @@
 version: '3.3'
 
 services:
-  db:
-   image: mysql:5.7
-   volumes:
-     - db_data:/var/lib/mysql
-   restart: always
-   environment:
-     MYSQL_ROOT_PASSWORD: gsviec
-     MYSQL_DATABASE: gsviec
-     MYSQL_USER: gsviec
-     MYSQL_PASSWORD: gsviec
-   ports:
-     - 3309:3306
-
-  php:
-   depends_on:
-     - db
-     - redis
-   image: gsviec/php:7.3-alpine
-   ports:
-     - "80:80"
-   volumes:
-    - ./:/app
-   environment:
-     DB_HOST: db:3306
-     DB_USER: root
-     DB_PASSWORD: gsviec
-     DB_NAME: gsviec
-     APPLICATION_ENV: local
-     REDIS_HOST: redis
-   env_file:
-      - .env
-
-  phpmyadmin:
-    restart: 'no'
-    image: phpmyadmin/phpmyadmin:fpm-alpine
-    depends_on:
-      - db
-    ports:
-      - "9090:80"
-    environment:
-      PMA_ARBITRARY: 1
-      PMA_HOST: db
-      PMA_PORT: 3306
-      PMA_USER: root
-      PMA_PASSWORD: gsviec
-
-  redis:
-    image: redis:alpine
-    ports:
-      - '63790:6379'
+    db:
+        image: mysql:5.7
+        volumes:
+            - db_data:/var/lib/mysql
+        restart: always
+        environment:
+            MYSQL_ROOT_PASSWORD: gsviec
+            MYSQL_DATABASE: gsviec
+            MYSQL_USER: gsviec
+            MYSQL_PASSWORD: gsviec
+        ports:
+            - 3309:3306
+    php:
+        depends_on:
+            - db
+            - redis-master
+        image: gsviec/video:v9
+        build: .
```

Comment:
remove it, we do not use Dockerfile image on that repo, in the future will be remove